### ISSUE TITLE
Depreciated Twig calls

### ISSUE THREAD

**Issue by dominickp** [1up-lab/OneupUploaderBundle#189]
I did a composer update and noticed some depreciated calls now associated with this bundle:

``` 
DEPRECATED - Using an instance of "Twig_Function_Method" for function "oneup_uploader_endpoint" is deprecated. Use Twig_SimpleFunction instead.
DEPRECATED - Using an instance of "Twig_Function_Method" for function "oneup_uploader_progress" is deprecated. Use Twig_SimpleFunction instead.
DEPRECATED - Using an instance of "Twig_Function_Method" for function "oneup_uploader_cancel" is deprecated. Use Twig_SimpleFunction instead.
DEPRECATED - Using an instance of "Twig_Function_Method" for function "oneup_uploader_upload_key" is deprecated. Use Twig_SimpleFunction instead.
DEPRECATED - Using an instance of "Twig_Function_Method" for function "oneup_uploader_maxsize" is deprecated. Use Twig_SimpleFunction instead.
```

**Comment by kissara** [comment]
<?php

namespace Oneup\UploaderBundle\Twig\Extension;

use Oneup\UploaderBundle\Templating\Helper\UploaderHelper;

class UploaderExtension extends \Twig_Extension
{
    protected $helper;

    public function __construct(UploaderHelper $helper)
    {
        $this->helper = $helper;
    }

    public function getName()
    {
        return 'oneup_uploader';
    }

    public function getFunctions()
    {
        return array(
            'oneup_uploader_endpoint'   => new \Twig_Function_Method($this, 'endpoint'),
            'oneup_uploader_progress'   => new \Twig_Function_Method($this, 'progress'),
            'oneup_uploader_cancel'     => new \Twig_Function_Method($this, 'cancel'),
            'oneup_uploader_upload_key' => new \Twig_Function_Method($this, 'uploadKey'),
            'oneup_uploader_maxsize'    => new \Twig_Function_Method($this, 'maxSize'),
        );
    }

    public function endpoint($key)
    {
        return $this->helper->endpoint($key);
    }

    public function progress($key)
    {
        return $this->helper->progress($key);
    }

    public function cancel($key)
    {
        return $this->helper->cancel($key);
    }

    public function uploadKey()
    {
        return $this->helper->uploadKey();
    }

    public function maxSize($key)
    {
        return $this->helper->maxSize($key);
    }
}

**Comment by giosh94mhz** [comment]
:+1: Since, Symfony 2.3 is now locked to bugfix only, I think it is safe to upgrade deprecated warning for 2.7 LTS version.

**Comment by bytehead** [comment]
Fixed by #194.
